.github/workflows/publish.yaml

```diff
@@ -0,0 +1,25 @@
+name: Publish
+on: [push]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2 # If you're using actions/checkout@v2 you must set persist-credentials to false in most cases for the deployment to work correctly.
+        with:
+          persist-credentials: false
+
+      - name: Install and Build 🔧 # convert the yaml schema to json version and docs, and publish on gh-pages
+        run: |
+          npm install js-yaml
```

Comment:
> but even that fails with a "command not found" error, even though the npm module I use there gets correctly installed.

does `--global` fix the issue. otherwise have no idea... don't have much expertise with javascript packaging.